frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
+      data-success-class="success"
```

Comment:
Yeah, but frontend developers (and even lazy people like me) don't tend to meddle with the middle of some JavaScript file to find out that they can actually customize the classes.